contributed_definitions/NXapm.nxdl.xml

```diff
@@ -447,41 +474,72 @@
                    microscopy) are accelerated towards a position-sensitive  
                    and time-of-flight taking detector system.  
                    Hence, there is no real probe/beam.  
-                 * Second, the specimen is the lens of an atom probe microscope.  
+                 * Second, the specimen is the lens of the microscope.  
             </doc>
             <field name="instrument_name" type="NX_CHAR">
                 <doc>
-                     Given name of the atom probe at the hosting institution. This is an 
+                     Given name of the atom probe at the hosting institution. This is an
                      alias. Examples could be LEAP5000, Raptor, Oxcart, one atom at a time,
                      etc.
                 </doc>
             </field>
-            <field name="location" optional="true" type="NX_CHAR">
+            <field name="location" type="NX_CHAR" optional="true">
                 <doc>
                      Location of the lab or place where the instrument is installed.
                      Using GEOREF is preferred.
                 </doc>
             </field>
-            <group type="NXmanufacturer"/>
+            <group type="NXmanufacturer">
```

Comment:
manufacturer in NXmanufacturer suggests that it is not about a component but its creator. Cannot we have a better name, like Product, Device, Component, Piece, Machine, or Fabrication because this definition is more about fabrication details, isn't it?

Reply:
NXfabrication